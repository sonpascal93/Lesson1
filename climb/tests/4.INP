100
7 9 0 7 9 -8 -5 7 -7 9 8 -7 4 2 5 4 5 0 -2 -8 5 -5 -4 1 5 -1 5 3 5 -2 -6 -6 0 0 8 8 7 4 8 2 -5 0 0 6 9 -8 7 0 -2 9 0 -2 -2 3 3 7 0 9 6 -2 9 2 -4 0 8 -8 -3 0 1 -4 0 0 -9 -7 6 -5 -3 0 -9 1 -4 -2 9 1 9 0 3 -3 3 -4 -4 -5 8 5 -7 7 7 -9 9 0 