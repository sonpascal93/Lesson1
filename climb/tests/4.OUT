32
6 9 12 14 16 17 21 25 27 29 35 36 37 38 40 42 43 48 51 57 64 68 71 72 78 82 88 90 91 92 95 98 